examples/aspnetcore-with-serilog/AspnetCoreExample.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
I think this should be part the main solution, because it references the Elastic.CommonSchema.Serilog.csproj, which has dependencies on other projects that may need to be compiled.

Reply:
Done.